app/json/slate_configs.json

```diff
@@ -101,7 +101,8 @@
           "493a5556-9800-449f-8f8c-c27bb6c8c810"
         ],
         "rankers": [
-          "top30-items"
+          "top30-items",
+          "firefox-thompson-sampling-1day"
```

Comment:
I actually think it's fine to leave this in here, and as a matter of fact, to add it to the three slates mentioned in the config _below_ this one.

Why: that's all 4 new tab slates and I'd like us to default to those all having consistent behavior unless we've got a good reason to make them different.